src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -18,14 +18,16 @@
 using Microsoft.AspNetCore.Protocols.Features;
 using Microsoft.AspNetCore.SignalR.Core;
 using Microsoft.AspNetCore.SignalR.Internal;
+using Microsoft.AspNetCore.SignalR.Internal.Formatters;
 using Microsoft.AspNetCore.SignalR.Internal.Protocol;
 using Microsoft.Extensions.Logging;
 
 namespace Microsoft.AspNetCore.SignalR
 {
     public class HubConnectionContext
     {
-        private static Action<object> _abortedCallback = AbortConnection;
+        private static readonly Action<object> _abortedCallback = AbortConnection;
+        private static readonly byte[] _successHandshakeResponseData = {(byte)'{', (byte)'}', TextMessageFormatter.RecordSeparator};
```

Comment:
Can we use the `HandshakeProtocol` to do this (in a static constructor if necessary)? That way we don't have a magic byte array floating around.

Reply:
HandshakeProtocol is in another assembly. Can either make the data public or internal and share internals with this assembly.

Or move it somewhere else inside this assembly.

Reply:
I mean call `HandshakeProtocol.WriteResponseMessage`. You do that elsewhere, so it should be safe to do here... It would have to be in a static function/ctor though.